examples/Simple_SQL_Single_Database_with_Private_Endpoint/main.tf

```diff
@@ -1,7 +1,10 @@
 provider "azurerm" {
   features {}
+  subscription_id = "01111111111110-11-11-11-11"
```

Comment:
This is not the standard that we decided, please update this.

Reply:
updated with new one